crates/turbopack-mdx/src/lib.rs

```diff
@@ -58,9 +87,35 @@ async fn into_ecmascript_module_asset(
         anyhow::bail!("Not able to read mdx file content");
     };
 
+    let jsx_runtime = if let Some(runtime) = &*transform_options.jsx_runtime.await? {
+        match runtime.as_str() {
+            "automatic" => Some(mdxjs::JsxRuntime::Automatic),
+            "classic" => Some(mdxjs::JsxRuntime::Classic),
+            _ => None,
```

Comment:
Perhaps we should error here if an unexpected string is provided (or make jsx_runtime an enum to start with?)

Reply:
There are a couple of different sources of existing runtime enum for the runtime so yes, this might need some update later.